meson_options.txt

```diff
@@ -25,3 +25,15 @@ option(
   type: 'boolean',
   value: false,
 )
+
+option(
+  'build_closed_source',
+  type: 'boolean',
+  value: false,
+)
+
+option(
+  'closed_source_dir',
+  type: 'string',
+  value: '',
+)
```

Comment:
Can we test `get_option('closed_source_dir')` for an empty string? That way, we can get rid of the above option `build_closed_source`.

Reply:
Off-course it is possible and also considered, I think it is more user-friendly to separate the true/false flag from the directory path. It's up to the tests code maintainer to decide.

Reply:
@maayanKash, I agree with @sriyerg here. I think it is more maintainable to just have to maintain one option. Could you please remove the boolean option, and just test for the empty string as @sriyerg suggests? Thanks!

Reply:
Sure, will do.